server/helpers/db.js

```diff
@@ -12,8 +12,30 @@ const getCourseById = course_id => {
     .from("courses")
     .where("course_id", "=", course_id);
 };
-
+const getOrganisations = () => {
```

Comment:
getOrganisations and getOrganisationsById don't need to be implemented as part of this task